.github/workflows/ci.yaml

```diff
@@ -61,6 +61,15 @@ jobs:
           VAULT_DEV_ROOT_TOKEN_ID: 227e1cce-6bf7-30bb-2d2a-acc854318caf
         ports:
           - 8200:8200
+      bao:
+        image: csatib02/openbao:dev
```

Comment:
lets use the forked version from https://github.com/bank-vaults/openbao-internal

Reply:
please make appropriate changes everywhere where this is needed

Reply:
We have an open issue about this, if there will be no images available when it is time to release, I will add the forked version everywhere:

issues:
https://github.com/bank-vaults/secrets-webhook/issues/27

Reply:
this is for when the bao is released officially. i would still use the `bank-vaults` bao release to ensure that images originate from trusted sources

Reply:
quay.io/openbao/openbao@sha256:a015ae0adb1af5b45b33632e29879ff87063d0878e9359584a50b2706e500e9a
```suggestion
        image: quay.io/openbao/openbao@sha256:a015ae0adb1af5b45b33632e29879ff87063d0878e9359584a50b2706e500e9a # 2.0.0-alpha20240329
```